File: Python/FilmBioskop.py
from Film import Film

# Derived class
class FilmBioskop(Film):
    def __init__(self, IdMedia: str, Judul: str, TahunRilis: int, Genre: str, Durasi: int, RatingUsia: str, FormatFilm: str, BahasaAudio: str, KualitasGambar: str):
        super().__init__(IdMedia, Judul, TahunRilis, Genre, Durasi, RatingUsia)
        self.__FormatFilm = str(FormatFilm)
        self.__BahasaAudio = str(BahasaAudio)
        self.__KualitasGambar = str(KualitasGambar)

    # Setter
    def setFormatFilm(self, FormatFilm: str) -> None:
        self.__FormatFilm = str(FormatFilm)

    def setBahasaAudio(self, BahasaAudio: str) -> None:
        self.__BahasaAudio = str(BahasaAudio)

    def setKualitasGambar(self, KualitasGambar: str) -> None:
        self.__KualitasGambar = str(KualitasGambar)

    # Getter
    def getFormatFilm(self) -> str:
        return self.__FormatFilm

    def getBahasaAudio(self) -> str:
        return self.__BahasaAudio

    def getKualitasGambar(self) -> str:
        return self.__KualitasGambar

File: Java/FilmBioskop.java
public class FilmBioskop extends Film {
    private String FormatFilm;
    private String BahasaAudio;
    private String KualitasGambar;

    public FilmBioskop() {
    }

    public FilmBioskop(String IdMedia, String Judul, int TahunRilis, String Genre, int Durasi, String RatingUsia, String FormatFilm, String BahasaAudio, String KualitasGambar) {
        super(IdMedia, Judul, TahunRilis, Genre, Durasi, RatingUsia);
        this.FormatFilm = FormatFilm;
        this.BahasaAudio = BahasaAudio;
        this.KualitasGambar = KualitasGambar;
    }

    // Setter
    public void setFormatFilm(String FormatFilm) {
        this.FormatFilm = FormatFilm;
    }

    public void setBahasaAudio(String BahasaAudio) {
        this.BahasaAudio = BahasaAudio;
    }

    public void setKualitasGambar(String KualitasGambar) {
        this.KualitasGambar = KualitasGambar;
    }

    // Getter
    public String getFormatFilm() {
        return this.FormatFilm;
    }

    public String getBahasaAudio() {
        return this.BahasaAudio;
    }

    public String getKualitasGambar() {
        return this.KualitasGambar;
    }
}